pyswervedrive/swervechassis.py

```diff
@@ -168,29 +155,20 @@ def update_odometry(self):
             velocity_outputs[i*2+1, 0] = velocity_y
```

Comment:
Now that we've removed the velocity odometry stuff, this is unused (along with a fair bit of related stuff in SweveModule). I wonder what the performance gains would be if we removed all that.

Reply:
I want to keep this in there for now - we may find that we still need it, eg if we have/want to go back to the BNO.

Reply:
This also will not be incurring a big performance penalty - the main gain comes from removing the least squares solving on velocity, and even that is a marginal speedup.